core/src/main/scala/org/http4s/rho/bits/HListToFunc.scala

```diff
@@ -5,11 +5,14 @@ import cats.Monad
 import org.http4s.rho.Action
 import shapeless.HList
 
+import scala.annotation.implicitNotFound
+
 /** Converter of an value of type F to the HList of type T
   *
  * @tparam T HList type of the incoming values
  * @tparam U type of element onto which T will be mapped
  */
+@implicitNotFound("No HListToFunc instance found. Most likely no ResultMatcher instance is available for the return type of (${U}). This can be caused by a lack of a matching EntityEncoder.")
```

Comment:
```suggestion
@implicitNotFound(
"""Could not bind route to action;
No HListToFunc instance found. 
  Most likely no ResultMatcher instance is available for the return type of 
    (${U}). 
  This can be caused by a lack of a matching EntityEncoder.""")
```

Yields:
```
[E] [E3] examples/src/main/scala/com/http4s/rho/swagger/demo/MyRoutes.scala:24:22
[E]      Could not bind route to action;
[E]      No HListToFunc instance found. 
[E]        Most likely no ResultMatcher instance is available for the return type of 
[E]          (Int => com.http4s.rho.swagger.demo.MyRoutes.Foo). 
[E]        This can be caused by a lack of a matching EntityEncoder.
[E]      L24:   GET / pathVar[Int] |>> { (i: Int) => Foo("a", "b")} 
[E]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```